stable/app/templates/deployment.yaml

```diff
@@ -110,6 +110,13 @@ spec:
           - containerPort: 55679
             name: zpages
             protocol: TCP
+          envFrom:
+            - configMapRef:
```

Comment:
Can we make this optional?

Reply:
@mabdh done, added `otelcollector.useEnvFromConfig` flag to use this feature (default: false)